web-registry/src/pages/Marketplace/Storefront/Storefront.tsx

```diff
@@ -39,9 +49,15 @@ export const Storefront = (): JSX.Element => {
   const [cardItems, setCardItems] = useState<Item[] | undefined>(undefined);
   const isBuyModalOpen = selectedSellOrder !== null;
 
+  const projectsInfosByHandleMap = normalizeProjectsInfosByHandleMap({
```

Comment:
Here we build a map that contains project:
- name
- handle
- classId
- classUrl